CONTRIBUTING.md

```diff
@@ -0,0 +1,244 @@
+3. **Relevant.** They should use game-relevant fluff and explain why what they're demonstrating is useful.
+4. **Minimal.** They should be no larger or more complex than is needed to meet their other goals.
+This reduces maintenance burden and improves clarity when used as a reference.
+
+When you add a new example, be sure to update `examples/README.md` with the new example.
+Use a generous sprinkling of keywords in your description: these are commonly used to locate a specific example using CTRL+F.
+
+More complex demonstrations of functionality are also welcome, but for now belong in community tutorials or template games.
+
+Check out [awesome-bevy](https://github.com/bevyengine/awesome-bevy) for a place to put your tutorials, tools, templates and crates!
+
+### Reviewing others work
+
+With the sheer volume of activity in Bevy's community, reviewing others work and helping it improve is one of the most valuable things you can do.
+You don't need to be an Elder Rustacean to be useful here: anyone can catch issues of missing tests, unclear docs, logic errors or so on.
+If you have unusual skills (e.g. advanced familiarity with `unsafe` code, rendering knowledge or web development experience) or personal experience with a problem, try to prioritize those areas to ensure we can get appropriate expertise where we need it.
+
+Focus on giving constructive, actionable feedback that results in real improvements to code quality or end-user experience.
+If you don't understand why an approach was taken, please ask!
+
+Small changes work well as comments or in-line suggestions on specific lines of codes.
+Larger changes deserve a comment in the main thread, or a pull request to the original author's branch (but mention that you've made one).
+When in doubt about a matter of architectural philosophy, refer back to **What we're trying to build** for guidance.
+
+Once you're happy with the work and feel you're reasonably qualified to assess quality in this particular area, leave your `Approved` review on the PR so we can mark it as `ready-for-cart`.
+
+There are a two main places you can check for new work to review:
+
+1. Pull requests on `bevy` and the `bevy-website` repos.
+2. [RFCs](https://github.com/bevyengine/rfcs), which need extensive thoughtful community input on their design.
+
+Official focus areas and work done by @cart go through this review process as well.
+Not even our project lead is exempt from reviews and RFCs!
+By giving feedback on this work (and related supporting work), you can help us make sure our releases are both high-quality and timely.
+
+Finally, if nothing brings you more satisfaction than seeing every last issue tagged and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
+As discussed in **How we're organized**, this is not intended to have a high bar.
```

Comment:
Can we make references like `**How we're organized**` clickable links? The headers should get anchor tags in Github.

There are a few more of these references down the line.